### PR TITLE
github: adds namespace uda

### PR DESCRIPTION
**Pull request by wookietreiber**
completely analogous to #503

### CODE REVIEW

**Comment by ryneeverett** [comment]
Needs a rebase.

**Comment by wookietreiber** [comment]
rebased

File: bugwarrior/services/github.py [hunk 1]
```diff
@@ -214,6 +219,7 @@ def to_taskwarrior(self):
             self.NUMBER: self.record['number'],
             self.CREATED_AT: created,
             self.UPDATED_AT: self.parse_date(self.record['updated_at'])
```
Comment (ryneeverett) [inline bugwarrior/services/github.py]: Syntax error. Needs a comma. :)

Comment (wookietreiber) [reply bugwarrior/services/github.py]: Alright, fixed.

**Pull request merged**